tracer/src/Datadog.Trace/AppSec/EventTrackingSdk.cs

```diff
@@ -49,7 +49,7 @@ internal static void TrackUserLoginSuccessEvent(string userId, IDictionary<strin
 
             var setTag = TaggingUtils.GetSpanSetter(span);
 
-            setTag(Tags.AppSec.EventsUsersLogin.SuccessTrack, bool.TrueString);
+            setTag(Tags.AppSec.EventsUsersLogin.SuccessTrack, "true");
```

Comment:
I think there are a bunch of tests expecting `"True"` at the moment that will need to be fixed

Reply:
And I assume we shall add a comment to avoid someone putting back `TrueString` in the future :). Or even better, enforcing it in a test